### PR TITLE
use getPCIDevices() to include missed videos cards

### PR DESCRIPTION
**Pull request by biji**
related to https://github.com/glpi-project/glpi-agent/pull/217

### CODE REVIEW

File: lib/GLPI/Agent/Task/Inventory/Linux/Videos.pm [hunk 1]
```diff
@@ -90,28 +91,38 @@ sub doInventory {
...
+            section => 'VIDEOS',
+            entry   => $video
+        );
 
-    if ($video->{MEMORY} && $video->{MEMORY} =~ s/kb$//i) {
-        $video->{MEMORY} = int($video->{MEMORY} / 1024);
-    }
-    if ($video->{RESOLUTION}) {
-        $video->{RESOLUTION} =~ s/@.*//;
+        $skipPci = $video->{PCIID};
```
Comment (g-bougard) [inline lib/GLPI/Agent/Task/Inventory/Linux/Videos.pm:117]: By now, you should not trust $video->{PCIID} as you have no guaranty it is set.

**Review by g-bougard — CHANGES_REQUESTED** [review]
Hi @biji 
this is better than previous PR.
Anyway you check PCIID which is not always set... sorry, this is not enough to avoid duplicate entries.

**Comment by g-bougard** [comment]
Maybe linux video support should completely be rewritten. I'm still not satisfied by the current implementation as for example we can't trust datas from the Xorg log file.

**Comment by biji** [comment]
using `lspci` should run on wider system, with wayland or xorg.. but can't detect memory size

**Comment by g-bougard** [comment]
Replaced by #233

**Pull request closed without merging**